docs/dapp/opl/README.mdx

```diff
@@ -0,0 +1,79 @@
+---
+description: Integrating privacy into dApps on other blockchains
+---
+
+import DocCard from '@theme/DocCard';
+import DocCardList from '@theme/DocCardList';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+
+# Oasis Privacy Layer (OPL)
+
+The Oasis Privacy Layer (OPL) is a powerful solution that enables developers
+to integrate privacy features into their decentralized applications (dApps)
+across multiple EVM-compatible networks. 
+
+- **Privacy-First**: OPL leverages the [Sapphire]'s privacy features to ensure
+that contract data and computation remains confidential.
+- **Cross-Chain Compatibility**: OPL is compatible with multiple blockchains
+through message bridging protocols, making it easy to integrate privacy
+regardless of the chain your dApp is built on.
+
+For more information about OPL and to catch the latest news, please visit the
+[official OPL page].
+
+[official OPL page]: https://oasisprotocol.org/opl
+[Sapphire]: https://oasisprotocol.org/sapphire
```

Comment:
This is actually a leftover from some presentation long ago. Linking the Sapphire chapter is better.

```suggestion
[Sapphire]: ../sapphire/README.mdx
```